app/src/main.cpp

```diff
@@ -495,6 +455,52 @@ namespace scitt
         .set_forwarding_required(ccf::endpoints::ForwardingRequired::Never)
         .install();
 
+      static constexpr auto get_entry_statement_path =
+        "/entries/{txid}/statement";
+      auto get_entry_statement = [this](
+                                   EndpointContext& ctx,
+                                   ccf::historical::StatePtr historical_state) {
+        SCITT_DEBUG("Get transaction historical state");
+        auto historical_tx = historical_state->store->create_read_only_tx();
+
+        auto entries = historical_tx.template ro<EntryTable>(ENTRY_TABLE);
+        auto entry = entries->get();
+        if (!entry.has_value())
+        {
+          throw BadRequestError(
+            errors::InvalidInput,
+            fmt::format(
+              "Transaction ID {} does not correspond to a submission.",
+              historical_state->transaction_id.to_str()));
+        }
+
+        auto cose_receipt = get_cose_receipt(historical_state->receipt);
+
+        // See https://datatracker.ietf.org/doc/draft-ietf-scitt-architecture/
+        // Page 16, 394 is the label for an array of receipts in the unprotected
+        // header
+        int64_t receipts = 394;
```

Comment:
there should be a static variable for this already called `COSE_HEADER_PARAM_SCITT_RECEIPTS`